static/benchmarks/css/competition.sass

```diff
@@ -23,9 +27,21 @@ section#competition
     font-weight: bold
     border-bottom: 1px solid $brain-score-color
 
+
 #stimuli-samples
   max-height: 325px
   overflow-y: hidden
 
   .stimulus_sample
     height: 50px
+
+.column
+  float: left
+  width: 33.33%
+  padding: 5px
+
+.row::after
+  content: ""
+  clear: both
+  display: table
```

Comment:
use bulma for this, see https://bulma.io/documentation/columns/basics/ or https://bulma.io/documentation/layout/tiles/